### PR TITLE
MGMT-8085

### PR DESCRIPTION
**Pull request by isaacdorfman**
# Added a migration for the ssh_public_key field

## Description
Added a migration for the ssh_public_key field in the clusters table.
Migrating from varchar(2048) to text.

## List all the issues related to this PR

- [ ] New Feature
- [x] Bug fix
- [ ] Tests
- [ ] Documentation
- [ ] CI/CD 

## What environments does this code impact?

- [ ] Automation (CI, tools, etc)
- [x] Cloud
- [ ] Operator Managed Deployments
- [ ] None

## How was this code tested?

- [ ] assisted-test-infra environment
- [ ] dev-scripts environment
- [ ] Reviewer's test appreciated
- [ ] Waiting for CI to do a full test run
- [x] Manual (Elaborate on how it was tested)
- [ ] No tests needed

## Assignees

/cc @
/cc @

## Checklist

- [ ] Title and description added to both, commit and PR.
- [ ] Relevant issues have been associated (see [CONTRIBUTING] guide)
- [ ] Reviewers have been listed
- [ ] This change does not require a documentation update (docstring, `docs`, README, etc)
- [ ] Does this change include unit-tests (note that code changes require unit-tests)

## Reviewers Checklist

- [ ] Are the title and description (in both PR and commit) meaningful and clear?
- [ ] Is there a bug required (and linked) for this change?
- [ ] Should this PR be backported?

[Kubernetes community documentation]: https://github.com/kubernetes/community/blob/master/contributors/guide/pull-requests.md#commit-message-guidelines
[CONTRIBUTING]: https://github.com/openshift/assisted-service/blob/master/CONTRIBUTING.md

### CODE REVIEW

**Comment by isaacdorfman** [comment]
@carbonin

**Comment by eranco74** [comment]
> @isaacdorfman please add a description and title.

/hold until :point_up:

File: internal/common/common_unitest_db.go [hunk 1]
```diff
@@ -47,7 +47,7 @@ func InitializeDBTest() {
 		oldResource.Close()
 	}
 	resource, err := pool.RunWithOptions(&dockertest.RunOptions{
-		Repository: "postgresql-12-centos7",
+		Repository: "quay.io/edge-infrastructure/postgresql-12-centos7",
```
Comment (rollandf) [inline internal/common/common_unitest_db.go:50]: Why is this needed in this PR?

**Comment by carbonin** [comment]
Closing in favor of https://github.com/openshift/assisted-service/pull/3200
